src/frontend/styles/components/writing.pcss

```diff
@@ -1,6 +1,6 @@
 .writing-header {
-  padding: 15px 0;
-  margin-top: calc(-1 * var(--layout-padding-vertical));
+  padding: 0 0 15px 0;
```

Comment:
looks like this changes irrelevan to this PR.

Reply:
It seems that I have corrected it and now it should be right. But I am not sure.